tests/api/test_opds_for_distributors.py

```diff
@@ -582,6 +584,71 @@ def test__add_format_data(self):
         # Undo the mock of SUPPORTED_MEDIA_TYPES.
         api.SUPPORTED_MEDIA_TYPES = old_value
 
+    def test_update_work_for_edition_returns_correct_license_pool(self):
+        # If there are two or more collections, `update_work_for_edition`
+        # should return the license pool for the right one.
+        data_source_name = "BiblioBoard"
+        _ = DataSource.lookup(self._db, data_source_name, autocreate=True)
+        data_source = DataSource.lookup(self._db, data_source_name, autocreate=True)
```

Comment:
Is this `lookup` run twice for a reason?

Reply:
Haha. Not a good one. Will fix.

Reply:
Fixed.